lib/execution_engine2/SDKMethodRunner.py

```diff
@@ -857,19 +871,31 @@ def check_workspace_jobs(self, workspace_id, projection=None):
 
     @staticmethod
     def _job_state_from_jobs(jobs):
+        """
+        Returns as per the spec file
+
+        :param jobs: MongoEngine Job Objects Query
+        :return: list of job states of format
+        Special Cases:
+        str(_id)
+        str(job_id)
+        float(created/queued/estimating/running/finished/updated/) (Time in MS)
+        """
         job_states = []
         for job in jobs:
             mongo_rec = job.to_mongo().to_dict()
             mongo_rec["_id"] = str(job.id)
             mongo_rec["job_id"] = str(job.id)
-            mongo_rec["created"] = str(job.id.generation_time)
-            mongo_rec["updated"] = str(job.updated)
+            mongo_rec["created"] = int(job.id.generation_time.timestamp()) * 1000
+            mongo_rec["updated"] = int(job.updated) * 1000
             if job.estimating:
-                mongo_rec["estimating"] = str(job.estimating)
+                mongo_rec["estimating"] = int(job.estimating) * 1000
+            if job.queued:
+                mongo_rec["queued"] = int(job.queued) * 1000
             if job.running:
-                mongo_rec["running"] = str(job.running)
+                mongo_rec["running"] = int(job.running) * 1000
             if job.finished:
-                mongo_rec["finished"] = str(job.finished)
+                mongo_rec["finished"] = int(job.finished) * 1000
```

Comment:
It will round timestamp to whole seconds. I don't know if it matters but we should do int(timestamp() * 1000).

Reply:
Thanks I fixed this